instrumentation/opentelemetry-instrumentation-botocore/tests/test_botocore_instrumentation.py

```diff
@@ -103,35 +103,36 @@ def test_s3_client(self):
             ),
         )
 
-    @mock_s3
```

Comment:
I think it would be better to [skip](https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions) this test:

``` python
    from pytest import mark
    ...
    @mock_s3
    @mark.skip(reason="This test is currently blocking the build, see #1088")
    def test_s3_put(self):
        ...
```

Reply:
Would it make much sense to import a new library `mark` just for this issue? We will potentially solve it soon anyways. Also, commenting makes it easier to find afterwards.

Reply:
I think it does, since this pytest feature is made for this particular purpose. I think it is easier to find this way because in this way the skipped test gets properly reported in the results. Nevertheless, this is not that important as to block the build, so I am approving this PR, if you prefer just to comment out the test that'ts fine by me :+1: